### DISCUSSION TITLE
How do I edit the default styles?

### DISCUSSION THREAD

**Discussion by ddm50** [GrapesJS/grapesjs#4949] [Q&A]
![image](https://user-images.githubusercontent.com/82258474/221601512-e4a46c3d-6370-40ca-ae02-0662493d5025.png)
Well, I need to provide some sort of default theme, have no idea how to do it and i cant find any answers or doc

**Comment by artf** [comment]
Here some of the options you have: 
https://github.com/GrapesJS/grapesjs/blob/45f23acf975d8ab09b60c095a3b405002b8c1658/src/editor/config/config.ts#L133-L154
